src/util_types/mutator_set/chunk.rs

```diff
@@ -311,4 +314,69 @@ mod tests {
 
         assert_eq!(chunk, decoded);
     }
+
+    #[ignore = "informative statistics"]
```

Comment:
Please add a comment describing the purpose of the test and any additional context in your head.